instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import dummyData from "../../dummy-data";
+import PostContainer from "./PostContainer";
+import SearchBar from "../SearchBar/SearchBar";
+ class PostsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      postData: [],
```

Comment:
you would normally want to have a separate postData on state to keep track of all of your posts. The way you have this set up is going back to the dummyData each time you filter, but you would want to have your postData be dynamic from dummyData. A good solution would be to use a separate filteredData in addition to the postData item, and show the filteredData list